src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void updateFruitStorage(FruitTransaction fruitTransaction,
+                                   FruitStorageDao fruitStorageDao) {
+        fruitStorageDao.increaseQuantity(fruitTransaction);
+    }
```

Comment:
The class `ReturnOperationHandler` should not directly call `increaseQuantity` on `FruitStorageDao`. Instead, it should perform any necessary checks and calculations before updating the storage. This would follow the Single Responsibility Principle, keeping the class focused on the 'return' operation logic.